test/cql-pytest/test_tools.py

```diff
@@ -696,18 +696,26 @@ def test_external_dir_autodetect_conf_dir_home_env(self, scylla_path, system_scy
 
 
 @pytest.fixture(scope="module")
-def scrub_schema_file():
```

Comment:
Wouldn't it be enough to change to `scope="function"`?

Reply:
We don't want to regenerate this file on every test, slowing down all tests. This file is read-only, once it is generated.

Reply:
Ah, makes sense